src/reducers/tests/actions/moxios_mock.jsx

```diff
@@ -0,0 +1,17 @@
+const Response = {
```

Comment:
The variable name should be in lowercase

Reply:
noted